pos_analytic_by_config/models/analytic_distribution_model.py

```diff
@@ -12,9 +12,10 @@ class AccountAnalyticDistributionModel(models.Model):
         help="Select a Point of Sale for which the analytic distribution will be used",
     )
 
-    def _get_distribution(self, vals):
+    def _get_distribution(self, new_vals):
+        new_vals = new_vals.copy()
```

Comment:
I meant that:

```suggestion
    def _get_distribution(self, vals):
        new_vals = vals.copy()
```

or you can reuse `vals` if you prefer, as it's in that moment a local variable.

Reply:
Yes, much better. Thanks!